### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by reyang**
1. introduce 4 issue templates
2. remove the old bug template
3. automatically tag issues based on category

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
-
-## Expected Behavior
-
-
-## Version Info 
-SDK Version :
-.NET Version :	 						
-How Application was onboarded with SDK(VisualStudio/StatusMonitor/Azure Extension) :
-OS :
-Hosting Info (IIS/Azure WebApps/ etc) :
```
Comment (TimothyMothra) [inline .github/ISSUE_TEMPLATE.md]: Can you please put these questions in the Bug template?
We need to know these details when investigating reported issues

Comment (TimothyMothra) [reply .github/ISSUE_TEMPLATE.md]: Oh, i see what you did. It's in the Describe writing prompt.
I'm okay with this, but if people don't follow the prompt we will need to put these questions back.

File: .github/ISSUE_TEMPLATE/documentation_issue.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation Issue
+about: Create a report to help us improve the documents
+labels: documentation
+---
+
+**Document URL.**
+What is the public documentation URL?
```
Comment (cijothomas) [inline .github/ISSUE_TEMPLATE/documentation_issue.md]: Can we ask docs issues to be opened in the docs repo directly?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+labels: bug
+---
+
+**Describe your environment.**
+Describe any aspect of your environment relevant to the problem, including your SDK version, platform, OS version, etc. If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on master.
```
Comment (cijothomas) [inline .github/ISSUE_TEMPLATE/bug_report.md]: We can ask more specific question  - .NET Core version, .NET Framework version? (value of `TargetFramework` in your .csproj)

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (reyang) [reply .github/ISSUE_TEMPLATE.md]: I've made it more explicit.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (reyang) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Added.

File: .github/ISSUE_TEMPLATE/documentation_issue.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+**Does the document belong to this repo?**
+If no, please file the issue against the document repo (e.g. https://github.com/MicrosoftDocs/azure-docs/).
```
Comment (cijothomas) [inline .github/ISSUE_TEMPLATE/documentation_issue.md]: Azure recommends opening issue by using the feedback button at the end of docs page, not directly through github. So lets add link to docs page here. and ask customers to open doc feedback there.
https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview

**Pull request merged**